.github/actions/eas-build/action.yml

```diff
@@ -46,7 +46,7 @@ runs:
 
         while [ $RETRIES -lt $MAX_RETRIES ]; do
           set +e
-          BUILD_ID=$(eas build -p $PLATFORM --profile $EAS_BUILD_PROFILE --message "$MESSAGE" --non-interactive --json --no-wait | jq -r ".[0].id")
+          BUILD_ID=$(eas build -p $PLATFORM --profile $EAS_BUILD_PROFILE --message "$MESSAGE" --non-interactive --json --no-wait --build-logger-level=debug | jq -r ".[0].id")
```

Comment:
should we keep this? that would increase fingerprint size which is not ideal for successful ci jobs. maybe we can just enable this when github action debugging is enabled, as you did in the continuous-deploy-fingerprint github action.

Reply:
Removed it for now. Added instructions when the test fails for how to re-add it. Doing it only during a GHA debug run is pretty tough: getting the value here is relatively easy, but getting the value inside of jest inside of a step on EAS build inside of EAS build inside of the GHA CI job is nontrivial.